packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -92,6 +93,10 @@ class Tooltip extends StatefulWidget {
 
   /// Whether the tooltip's [message] should be excluded from the semantics
   /// tree.
+  ///
+  /// Defaults to false. A tooltip will add a [Semantics.label] that is set to
+  /// [Tooltip.message]. Set this property to true if a custom
```

Comment:
I think you mean ... true if the app is going to provide its own custom semantics label?

Reply:
Yes! I'll update that